code/modules/codex/categories/category_materials.dm

```diff
@@ -69,12 +69,23 @@
 			else
 				material_info += "It is ~[comparison] times more durable than steel."
 
-			if(LAZYLEN(mat.chemical_makeup))
+			if(LAZYLEN(mat.dissolves_into))
 				var/chems = list()
-				for(var/chemial in mat.chemical_makeup)
-					var/decl/material/R = chemial
-					chems += "[initial(R.name)] ([mat.chemical_makeup[chemial]*100]%)"
-				material_info += "The following chemicals can be extracted from it:<br> [english_list(chems)]"
+				for(var/chemical in mat.dissolves_into)
+					var/decl/material/R = chemical
+					chems += "[initial(R.name)] ([mat.dissolves_into[chemical]*100]%)"
+
+				var/solvent_needed
+				if(mat.dissolves_in <= MAT_SOLVENT_NONE)
+					solvent_needed = "any liquid"
+				else if(mat.dissolves_in <= MAT_SOLVENT_MILD)
+					solvent_needed = "a mild solvent, like water"
+				else if(mat.dissolves_in <= MAT_SOLVENT_MODERATE)
+					solvent_needed = "a moderately strong solvent, like acetone"
+				else if(mat.dissolves_in <= MAT_SOLVENT_STRONG)
+					solvent_needed = "a strong solvent, like sulphuric acid"
+
+				material_info += "The following chemicals can be extracted from it with [solvent_needed] solvent:<br> [english_list(chems)]"
 
```

Comment:
add solvent properties of the chem itself too pls

Reply:
Done.